apps/frontend/src/components/template/QuestionPicker.tsx

```diff
@@ -141,9 +142,15 @@ export const QuestionPicker = (props: QuestionPickerProps) => {
       return true;
     }
 
-    const textMatch = question.question
-      .toLowerCase()
-      .includes(questionFilter.searchText.toLowerCase());
+    const textMatch =
+      question.dataType == 'EMBELLISHMENT'
```

Comment:
Maybe it is better to use `DataType.EMBELLISHMENT` here instead of hardcoding it directly